src/overlays/actors/ovl_En_Kaizoku/z_en_kaizoku.h

```diff
@@ -12,29 +12,32 @@ typedef void (*EnKaizokuActionFunc)(struct EnKaizoku*, PlayState*);
 #define KAIZOKU_GET_EXIT_INDEX(thisx) ((thisx)->params & 0x3F)
 #define KAIZOKU_GET_SWITCH_FLAG(thisx) (((thisx)->params >> 6) & 0x7F)
 
+// used for text and for outfit color
+#define KAIZOKU_GET_TYPE(this) ((this)->picto.actor.world.rot.z)
+
 #define KAIZOKU_SWITCH_FLAG_NONE 0x7F
 
 typedef enum EnKaizokuAnimation {
-    /*  0 */ EN_KAIZOKU_ANIM_0,
-    /*  1 */ EN_KAIZOKU_ANIM_1,
-    /*  2 */ EN_KAIZOKU_ANIM_2,
-    /*  3 */ EN_KAIZOKU_ANIM_3,
-    /*  4 */ EN_KAIZOKU_ANIM_4,
-    /*  5 */ EN_KAIZOKU_ANIM_5,
-    /*  6 */ EN_KAIZOKU_ANIM_6,
-    /*  7 */ EN_KAIZOKU_ANIM_7,
-    /*  8 */ EN_KAIZOKU_ANIM_8,
-    /*  9 */ EN_KAIZOKU_ANIM_9,
-    /* 10 */ EN_KAIZOKU_ANIM_10,
-    /* 11 */ EN_KAIZOKU_ANIM_11,
-    /* 12 */ EN_KAIZOKU_ANIM_12,
-    /* 13 */ EN_KAIZOKU_ANIM_13,
-    /* 14 */ EN_KAIZOKU_ANIM_14,
-    /* 15 */ EN_KAIZOKU_ANIM_15,
-    /* 16 */ EN_KAIZOKU_ANIM_16,
-    /* 17 */ EN_KAIZOKU_ANIM_17,
-    /* 18 */ EN_KAIZOKU_ANIM_18,
-    /* 19 */ EN_KAIZOKU_ANIM_MAX
+    /*  0 */ KAIZOKU_ANIM_FIGHTING_IDLE,
+    /*  1 */ KAIZOKU_ANIM_UNUSED_CONVERSATION, // from OOT: giving player membership card?
+    /*  2 */ KAIZOKU_ANIM_UNUSED_JUMP, // from OOT: replaced with _LAND
+    /*  3 */ KAIZOKU_ANIM_SIDESTEP,
+    /*  4 */ KAIZOKU_ANIM_WALK,
+    /*  5 */ KAIZOKU_ANIM_DAMAGE,
+    /*  6 */ KAIZOKU_ANIM_UNUSED_DEFEAT, // from OOT: replaced with _DEFEAT
```

Comment:
```suggestion
    /*  6 */ KAIZOKU_ANIM_UNUSED_DEFEAT, // from OOT: replaced with `KAIZOKU_ANIM_DEFEAT`
```
I would prefer to write out the whole thing, especially since this animation *already* contains the string `_DEFEAT` in it.